test/e2e/spec/eid_service/main_spec.js

```diff
@@ -0,0 +1,39 @@
+describe("eID-Service: The general flow", function() {
+
+    var frisby = require('frisby');
+    var URL = 'https://' + process.env.BOILERPLATE_DOMAIN + '/api/eIdService/getTcToken';
+
+    beforeAll(function() {
+        frisby.addExpectHandler('tc token', function(response) {
+            let xml = response._body.replace(/\n/g, "").replace(/\r/g, "").replace(/ /g, "");
+
+            expect(xml).toEqual(jasmine.stringMatching(/^<TCTokenType><ServerAddress>.*<\/Binding><\/TCTokenType>$/));
+        });
+    });
+
+    it("is working fine", function(done) {
```

Comment:
The description here should be a little more descriptive. I advise to split this test case into several expectation handlers to make it more readable:

    frisby.get(....getTcTokenUrl....)
      .expect('status', 302)
      .expect('tcToken retrievable')

where the new `tcToken retrievable` expectation handlers implements most of the actual testing.

With #55 in mind, this should be compatible with [chakram](http://dareid.github.io/chakram/jsdoc/module-chakram-expectation.html) by extending [chai](http://chaijs.com/api/bdd/).